pkg/platform/aws/template.go

```diff
@@ -99,7 +99,7 @@ module "aws-{{.Config.ClusterName}}" {
 
 {{ range $index, $pool := .Config.WorkerPools }}
 module "worker-pool-{{ $index }}" {
-  source = "../lokomotive-kubernetes/aws/flatcar-linux/kubernetes/workers"
+  source = "../terraform-modules/aws/flatcar-linux/kubernetes/workers"
```

Comment:
Seems like we should be using constant for that as well (but perhaps out of scope :smile:)

Reply:
This would be a new constant as this path is the path *on disk* rather than the virtual "in-memory" asset path. I think we can look into this but IMO it doesn't have to be in this PR.

Reply:
I'll mark as resolved, nothing to take action on.